android/uhabits-core/src/test/java/org/isoron/uhabits/core/utils/DateUtilsTest.java

```diff
@@ -164,6 +164,7 @@ public void testTruncate_year()
     @Test
     public void testMillisecondsUntilTomorrow() throws Exception
     {
+        DateUtils.setFixedTimeZone(TimeZone.getTimeZone("GMT"));
```

Comment:
Was this required to make the tests pass in your computer? It makes sense, but I wonder why we haven't run into any issues before.

Reply:
because after this change `MillisecondsUntilTomorrow` goes through  `getUpcomingTimeInMillis` which is time zone dependent